examples/bri-3/src/bri/identity/bpiAccounts/accounts.module.ts

```diff
@@ -32,4 +32,5 @@ export const QueryHandlers = [
   ],
   exports: [BpiAccountAgent, BpiAccountStorageAgent],
 })
+// TODO: Move this under a separate folder 'state' instead of 'identity'
```

Comment:
Is this already an open issue?

Reply:
@Therecanbeonlyone1969  This code was copied from Ognjen's PR. This question has been addressed in PR #732